lib/prophet/prophet.rb

```diff
@@ -164,66 +164,70 @@ def pull_requests
   # - the pull request hasn't been used for a run before.
   # - the pull request has been updated since the last run.
   # - the target (i.e. master) has been updated since the last run.
+  # - the pull request does not originate from a fork (to avoid malicious code execution on CI machines)
```

Comment:
Wondering how complicated we want to do this, but maybe it should be an optional feature?